test/ejabberd_sm_SUITE.erl

```diff
@@ -217,7 +217,26 @@ kv_can_be_updated_for_session(C) ->
     when_session_info_stored(U, S, R, {key2, newval}),
     when_session_info_stored(U, S, R, {key2, override}),
 
-    [#session{sid = Sid, info = [{key1, val1}, {key2, override}]}]
+    ?assertMatch([#session{sid = Sid, info = [{key1, val1}, {key2, override}]}],
+                 ?B(C):get_sessions(U, S)).
+
+kv_can_be_removed_for_session(C) ->
```

Comment:
This is not being tested, it wasn't added to the `tests()` list: https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/2566/10426/small_tests.2218/small/ct_run.mongooseim@localhost.2019-12-13_10.05.09/lib.mongooseim.logs/run.2019-12-13_10.05.10/suite.log.html